### PR TITLE
regenerate dlp v2 library

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: dlp/README.rst [hunk 1]
```diff
@@ -61,38 +63,15 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install google-cloud-dlp
 
-Preview
```
Comment (crwilcox) [inline dlp/README.rst]: I can adjust the synth script to not move this file. this example might be useful.

File: dlp/nox.py [hunk 2]
```diff
@@ -18,35 +18,9 @@
...
-        '--quiet',
-        '--cov=google.cloud.dlp_v2',
-        '--cov-append',
-        '--cov-config=.coveragerc',
-        '--cov-report=',
-        '--cov-fail-under=97',
-        os.path.join('tests', 'unit'),
-        *session.posargs
-    )
+    return unit(session, 'default')
```
Comment (tseaver) [inline dlp/nox.py]: I don't understand why we are dropping the `system` / `lint` `cover` bits, nor why this library would be different than any of our other ones.

Comment (crwilcox) [reply dlp/nox.py]: this is the template that comes from our generator. I can revert this part since it seems to be dropping things.

**Comment by tseaver** [comment]
Are we not generating / creating any tests?

**Comment by crwilcox** [comment]
@tseaver, tests are generated but there aren't any changes.

**Comment by tseaver** [comment]
@crwilcox That doesn't seem to bode well for our coverage story, given that not all the changes are docs-only.

**Pull request merged**